api-guidelines/global/json/canonical-data-types/rules/should-use-common-otto-decimal-format.md

```diff
@@ -0,0 +1,24 @@
+---
+id: R100079
+---
+
+# SHOULD use common `otto:decimal` format
+
+When a business case processing JSON properties calls for a higher precision as available by using floats by default, using type `number` with format `otto:decimal` allows for custom conversion logic based on application programming language and not relying on [OpenAPI `number` data type][openapi-data-types] being automatically converted to `float` or `double`.
+
+Examples for correct representations as monetary amounts (in EUR):
```

Comment:
As this section is VERY close to Zalando...does their license this or do we need to come up with own examples and constraints? 

Furthermore, I feel a bit lost and would love an introductory sentence combining the decimal format and the examples.